source/_integrations/griddy.markdown

```diff
@@ -0,0 +1,65 @@
+---
+title: Griddy
+description: Instructions on how to integrate griddy prices into Home Assistant.
+logo: griddy.png
```

Comment:
Our logo's have moved to a different repository.

<https://github.com/home-assistant/brands>

Please open up a PR on that repository, adding the logo's.
Next, link it up with this PR and remove the `logo` frontmatter and file.

Reply:
Hopefully I got this right :)